mivs/automated_emails.py

```diff
@@ -51,6 +51,8 @@ def __init__(self, *args, **kwargs):
 MIVSEmail(IndieGame, 'MIVS December Updates: Hotels and Magfest Versus!', 'december_updates.txt',
           lambda game: game.confirmed)
 
+MIVSEmail(IndieGame, 'REQUIRED: Pre-flight for MIVS due by midnight, January 2nd', 'game_preflight.txt', lambda game: game.confirmed)
```

Comment:
Nice! Later on you may also want to add a `when` tag (like the email below has) so this doesn't get sent out immediately. On the other hand, since emails are based on approval, you can always just wait to approve this until the appropriate time.